demo/src/layouts/MainLayout.js

```diff
@@ -24,8 +24,8 @@ const MainLayout = (props) => {
       <SEO location={location} />
       <GlobalHeader />
       <MobileHeader>
-        {nav.map((page) => (
-          <NavItem key={page.url} page={page} />
+        {nav.map((page, index) => (
+          <NavItem key={`mobile${index}${page}`} page={page} />
```

Comment:
does it make sense for this to also use page.url (`mobile${index}${page.url}`)? instead of the whole page object

Reply:
I think we really only need the index, but I can add that url to make it more specific.